guides/common/modules/proc_importing-into-the-library-environment-from-a-web-server.adoc

```diff
@@ -0,0 +1,29 @@
+[id="Importing_into_the_Library_Environment_from_a_web_server_{context}"]
+= Importing into the Library Environment from a web server
+
+You can import exported Library content directly from a web server into the Library lifecycle environment of an organization on another {ProjectServer}.
+For more information about exporting contents from the Library environment, see xref:Exporting_the_Library_Environment_{context}[].
+
+.Prerequisites
+* The exported files must be in a syncable format.
+* The exported files must be accessible via HTTP/HTTPS.
+* If there are any Red Hat repositories in the exported content, the importing organization's manifest must contain subscriptions for the products contained within the export.
+* The user importing the Content View version must have the _Content Importer_ Role.
+*
+.Procedure
+. Select the organization into which you want to import.
```

Comment:
Is this necessary?

Reply:
Reworded it to `Identify the Organization that you wish to import into.` as it is in other sections.